src/illumidesk/authenticators/authenticator.py

```diff
@@ -72,10 +72,10 @@ async def setup_course_hook(
     # register the user (it doesn't matter if is a student or instructor) with her/his lms_user_id in nbgrader
     await jupyterhub_api.add_user_to_nbgrader_gradebook(course_id, username, lms_user_id)
     # TODO: verify the logic to simplify groups creation and membership
-    if user_role == 'Student' or user_role == 'Learner':
+    if user_role in ('Student', 'Learner'):
         # assign the user to 'nbgrader-<course_id>' group in jupyterhub and gradebook
         await jupyterhub_api.add_student_to_jupyterhub_group(course_id, username)
-    elif user_role == 'Instructor':
+    elif user_role in ('Instructor', 'urn:lti:role:ims/lis/TeachingAssistant'):
```

Comment:
We've gotten requests for other roles to have access to the grader group, so perhaps with worth the effort to avoid having to send a PR for every use-case.

Reply:
In this case I added a constant too (`DEFAULT_ROLE_NAMES_FOR_INSTRUCTOR`) and a helper method that reads a new envvar `EXTRA_ROLE_NAMES_FOR_INSTRUCTOR` to help us to extend the default values